### ISSUE TITLE
Need weighted softmax loss for semantic segmentation of imbalanced data

### ISSUE THREAD

**Issue by wdx04** [dmlc/gluon-cv#644]
Hi,
I have a semantic segmentation dataset where background occupies more than 99% of the image pixels. This is nearly impossible to train without the ability to specify the loss weight for each object class.
After training, the model always outputs an all-background result which has >99% pixel accuracy but 0% IoU.
Could you please added class weighting to the SoftmaxCrossEntropyLoss class?
